sdk/nodejs/lambda/lambdaMixins.ts

```diff
@@ -317,11 +317,20 @@ export class CallbackFunction<E, R> extends LambdaFunction {
             ...args,
             code: new pulumi.asset.AssetArchive(codePaths),
             handler: serializedFileNameNoExtension + "." + handlerName,
-            runtime: args.runtime || runtime.NodeJS8d10Runtime,
+            runtime: args.runtime || runtime.NodeJS10dXRuntime,
             role: role.arn,
             timeout: args.timeout === undefined ? 180 : args.timeout,
         };
 
+        // If there is no custom Runtime argument being passed to the user
+        // then we should add "runtime" to the ignoreChanges of the CustomResourceOptions
+        // This is because as of 12/16/19, we upgraded the default NodeJS version from 8.x to 10.x
+        // We don't want to force recreation of user defined Lambdas because of this change
+        if (!args.runtime) {
+            opts.ignoreChanges = opts.ignoreChanges || []
+            opts.ignoreChanges.push("runtime")
+        }
```

Comment:
We actually have `pulumi.mergeOptions` which takes care of this - and also avoids having to modify the user-provided `opts` in place which is probably technically incorrect.  

```ts
super(name, functionArgs, pulumi.mergeOptions(opts, { ignoreChanges: ["runtime"] }));
```

Reply:
Nice :)